.ci/aws/aws_ofi_nccl_pr_ci.yaml

```diff
@@ -6,5 +6,6 @@ cluster:
 testing:
   test_target: aws-ofi-nccl
   test_type: pr
+  aws_ofi_nccl_build_type: debug
```

Comment:
Where's the code that actually passes `--enable-debug` to plugin's configure?

Reply:
It's in our internal repository